web/Controllers/AbstractController.cs

```diff
@@ -11,4 +15,27 @@ protected BricksAndHeartsUser GetCurrentUser()
 
         return (BricksAndHeartsUser)User.Identity;
     }
+
+    protected void FlashRequestSuccess(ILogger logger, BricksAndHeartsUser user, string requestAction)
+    {
+        FlashMessage(logger,
+            ($"Successfully {requestAction} for user {user.Id}",
+                "success",
+                $"Successfully {requestAction}"));
+    }
+
+    protected void LoggerAlreadyAdminWarning(ILogger logger, BricksAndHeartsUser user)
+    {
+        FlashMessage(logger,
+            ($"User {user.Id} already an admin",
+                "danger",
+                "Already an admin"));
+    }

```

Comment:
Is anything other than the `AdminController` going to use this method? If not I would keep it there. The other two methods make sense to commonise to me though :)

Reply:
I think you're right, and it can easily be taken out again if it is required though